python/type_check_error.py

```diff
@@ -44,6 +44,12 @@ def display_type(n_type):
 	elif isinstance(n_type, tuple):
 		return Fore.YELLOW + ' -> '.join(n_type) + Style.RESET_ALL
 	elif isinstance(n_type, list):
+		try:
```

Comment:
Why would this be in a `try` block? Doesn't seem like it'd raise an exception

Reply:
it does raise an index out of bounds error, but I think we might be able to remove it because that was caused by another bug.

Reply:
Oh because of empty lists, they cause problems